frontend/src/components/TimeTravel.tsx

```diff
@@ -28,6 +38,17 @@ export const TimeTravel = ({
     setCurrentWeek(weekNumber(nextDate));
   };
 
+  const CustomDatePickerInput = forwardRef(({ value, onClick }, ref) => (
+    <label onClick={onClick} className="header-time-travel" ref={ref}>
+      Week {`${currentWeek}`}
+    </label>
```

Comment:
Typecript throws two errors in this part and I haven't managed to solve them yet. They don't break anything so I think it's fine to ignore them for now, but we should not forget about them.